src/fastoad/models/performances/mission/segments/base.py

```diff
@@ -133,6 +134,8 @@ class AbstractFlightSegment(IFlightPart, ABC):
     # To be noted: this one is not a dataclass field, but an actual class attribute
     _attribute_units = dict(reference_area="m**2", time_step="s")
 
+    start: FlightPoint = field(default=MANDATORY_FIELD, init=False)
```

Comment:
As you can see, `_attribute_units` is not a dataclass field, but a class attribute (and private). Please keep it in last position.